pkg/gardenlet/controller/shoot/shoot_control.go

```diff
@@ -23,8 +23,9 @@ import (
 
 	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
 	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
-	gardencorev1beta1helper "github.com/gardener/gardener/pkg/apis/core/v1beta1/helper"
+	v1beta1helper "github.com/gardener/gardener/pkg/apis/core/v1beta1/helper"
```

Comment:
Why was this changed? 👀

Reply:
gimps. Why have the import above be `v1beta1constants` and not `gardencorev1beta1constants`?

Reply:
I don't know, there is no policy I think, just "soft conventions". The constants pkg was always called `v1beta1constants` and the helper pkg was always called `gardencorev1beta1helper`. Anyways, I was just wondering about this (apparently unrelated) change.

Reply:
I run the formatter on all files I touch in a PR, and I thought that this change was surely uncontroversial, as it just applies the convention consistently. ;)